src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
+@import 'variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  width: $container-size;
+  padding: $container-padding;
+
+  display: flex;
+  gap: $space-between-box;
```

Comment:
Ensure that the variable `$space-between-box` is defined. This variable is used to set the gap between calendar days.